src/new-apps/components/AppDetailsPage/DataPrivacyCard.tsx

```diff
@@ -0,0 +1,46 @@
+import CardTitle from "@dashboard/components/CardTitle";
+import ExternalLink from "@dashboard/components/ExternalLink";
+import Skeleton from "@dashboard/components/Skeleton";
+import { Card, CardContent } from "@material-ui/core";
+import React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+
+import messages from "./messages";
+import { useStyles } from "./styles";
+
+interface DataPrivacyCardProps {
+  dataPrivacyUrl?: string | null;
+  loading: boolean;
+}
+
+const DataPrivacyCard: React.FC<DataPrivacyCardProps> = ({
+  dataPrivacyUrl,
+  loading,
+}) => {
+  const classes = useStyles();
+  const intl = useIntl();
+
+  if (!dataPrivacyUrl) {
+    return null;
+  }
```

Comment:
**suggestion**: I don't know exact implementation details but to me it looks like this component won't render skeletons when in loading state
```suggestion
  if (dataPrivacyUrl === null) {
    return null;
  }
```